Bio/SeqFeature.py

```diff
@@ -1772,20 +1808,23 @@ def position(self):
 
     @property
     def extension(self):
-        """Legacy attribute to get extension (zero) as integer (OBSOLETE)."""
+        """Legacy attribute to get zero as integer (OBSOLETE)."""
         return 0
 
     def __repr__(self):
-        """A string representation of the location for debugging."""
+        """Represent the location as a string for debugging."""
         return "%s(%i)" % (self.__class__.__name__, int(self))
 
     def __str__(self):
+        """Return a representation of the AfterPosition object (with python counting)."""
         return ">%s" % self.position
 
     def _shift(self, offset):
+        """Legacy attribute to get extension (zero) as integer (PRIVATE)."""
```

Comment:
Copy and paste error? See the other ``_shift`` methods where you had a sensible docstring, e.g.

``Return a copy of the position object with its location shifted (PRIVATE).``